### PR TITLE
Makes the hypnoflash and hypnoflashbang role-restricted to the Psychologist.

### PR DESCRIPTION
**Pull request by uomo91**
## About The Pull Request

This role-restricts the hypnoflash and hypnoflashbang to the Psychologist in the traitor uplink.

## Why It's Good For The Game

The hypnoflash is not a very commonly purchased item, in large part I assume because it's not an efficient killing tool. It's a liability and a bit of monkeying around to even try hypnoflashing somebody, and if you succeed it's a further liability on if the person you flashed will actually play along and obey or just rat you out, or screw up your plan/be incompetent if they do obey. If your objective is greentexting at all cost or murderboning, chances are you will not touch this item with a ten foot pole.

What the hypnoflash IS great for, is memes, funny things, and roleplay. And so, who better to give this item to than an RP job which attracts those type of players to begin with? In particular one for who it makes perfect RP sense for them to be hypnotizing people. I feel this makes at least as much sense as the current role-restrictions on the brainwashing surgery disk.

## Changelog
:cl:
tweak: The hypnotic flash and hypnotic flashbang are now role-restricted to traitor Psychologists.
/:cl:

### CODE REVIEW

**Comment by LordSaladin** [comment]
what

**Comment by ATH1909** [comment]
It's a very commonly purchased item by me, thank you very much.

And see? This is exactly the sort of shit that I said would happen if Psychologist got merged. Now a really cool traitor item is gonna become locked behind a curator/lawyer-like job, which means that it'll be used even _less_ often.

**Comment by ArcaneDefence** [comment]
Iunno. I've had fun with hypnoflash as a chef, because CQC has a very reliable sleep to it. 
Anyone that gets a hold of the bz from xenobio can then go around with a flamethrower of it and hypnotize people in the hallways with ease

**Comment by ATH1909** [comment]
> Iunno. I've had fun with hypnoflash as a chef, because CQC has a very reliable sleep to it.
> Anyone that gets a hold of the bz from xenobio can then go around with a flamethrower of it and hypnotize people in the hallways with ease

Or spread a hallucination/drowsiness virus as a virologist, or use a hypospray of mindbreaker toxin as a CMO, or...

You get the point, I believe.

**Comment by ATH1909** [comment]
"The hypnoflash is not a very commonly purchased item, in large part I assume because it's not an efficient killing tool."

hm yes this traitor item doesn't kill people that means no one cares about it and I should restrict it to only being purchaseable by a specific RP job

**Comment by XDTM** [comment]
The issue is that even if psychologist is the ideal role for these items, _restricting_ them will make them appear less often anyway, and personally i like to see rp-oriented traitor tools as often as possible compared to the murderboney standard.

If you want to promote psychologists using hypnosis (which they're already incentivized to do by starting with the necessary materials), a discount system would probably be ideal, although that would require some framework around it.

The brainwashing surgery disk, in my opinion, is restricted mostly because in most of non-doctor traitor rounds it's just going to clutter the list.

**Comment by Shadowflame909** [comment]
This is like making the e-sword chaplain only because that man knows the ways of the force.

We need additions uomo91. Please don't lock the little non-mass murderbone content that traitors have behind a job only antags will get a kick out of because of a new secret room.

**Comment by trollbreeder** [comment]
image
Uh oh.

**Comment by uomo91** [comment]
Hey, the plan succeeded!

**Comment by ATH1909** [comment]
> Hey, the plan succeeded!

But I _do_ actually use it. I'm pretty sure I have screenshots somewhere of it being in my list of purchased traitor items, if you'd like for me to dig them up.

**Comment by Shadowflame909** [comment]
Let me tell you about my friend the sleepy pen and how it's gotten me to brainwash some top tier powergamers like lexia and the like.

**Comment by uomo91** [comment]
> And see? This is exactly the sort of shit that I said would happen if Psychologist got merged. Now a really cool traitor item is gonna become locked behind a curator/lawyer-like job, which means that it'll be used even _less_ often.

Every role-restricted traitor item is locked behind a specific job people may or may not want to play as independent of having access to said item, really failing to see how this is different.

In the Psych PR you said:

> I like this job/this PR (and the moth plushie placement there is just top tier), but I'm worried about what will come after it. People are gonna want to add some psychology/advanced trauma mechanics for this job to interact with like what #48403 tried to do, and that scares me.

The main criticism of Psychologist so far has been _the lack_ of mechanics and unique content, here is one simple way of giving the job something unique as well as perfectly fitting, and now the sky is falling apparently. Curious.

> If anything, we should be removing the role restrictions on the brainwashing surgery disk, or putting all advanced surgeries on that surgery disk or something.

Well at least you're consistent.

**Comment by wesoda25** [comment]
Complaining about making it psych only just because you liked using it with x job because it was easy to cheat with said job isn’t really a counter, you just appear biased. The potential of this item consistently outshadowed its predecessor: the brainwashing surgery disk. Hopefully we’ll see that used more often now, albeit in its limited capacity as a medbay only option. 

In that same vein I guess I’m biased because I thought the hypno flash, and especially hypno flashbang were dumb editions, and they seem more rational now. w/e

**Comment by itseasytosee** [comment]
I don't see how in any way this could improve the state of the hypnoflash. It's not the most common item and most people don't see the impact it has on the game unless of course, you are the person being hypnotized or the traitor.
>What the hypnoflash IS great for, is memes, funny things, and roleplay.

And why take that away from everyone else? Do you expect that phyc revs will feel obligated to use it, and that will make it a more used and fun item?

**Comment by ATH1909** [comment]
> Complaining about making it psych only just because you liked using it with x job because it was easy to cheat with said job isn’t really a counter, you just appear biased. The potential of this item consistently outshadowed its predecessor: the brainwashing surgery disk. Hopefully we’ll see that used more often now, albeit in its limited capacity as a medbay only option.
> 
> In that same vein I guess I’m biased because I thought the hypno flash, and especially hypno flashbang were dumb editions, and they seem more rational now. w/e

That's because the brainwashing surgery disk is underpowered and really should be buffed at some point.

> > I like this job/this PR (and the moth plushie placement there is just top tier), but I'm worried about what will come after it. People are gonna want to add some psychology/advanced trauma mechanics for this job to interact with like what #48403 tried to do, and that scares me.
> 
> The main criticism of Psychologist so far has been _the lack_ of mechanics and unique content, here is one simple way of giving the job something unique as well as perfectly fitting, and now the sky is falling apparently. Curious.

Hey, I haven't been contradicting myself there, the very thing you quoted explicitly said that I was worried about what negative changes would be made to the game in the name of _adding_ more mechanics for the psychologist role.

Also, giving a job unique antag items (or rather, making antag items exclusive to only one job) isn't going to help solve that "_lack_ of mechanics and unique content" problem for non-antag psychologists. Sure, antag psychologists will get to have fun with a hypnotic flash, but the non-antag psychologists will be bored as shit. This kind of thinking was what lead to oranges eventually declaring an antag freeze for a few months; people kept trying to fill in holes of content with antag-only items. If anything, you should be adding a new and unique mechanic for psychologists that can be used for both good AND evil (and still be usable/accessible if you aren't an antag).

**Comment by uomo91** [comment]
Lol, trying to spin this as me centering the job around the antag experience is a creative take.

**Comment by bluetshirtguy** [comment]
EH, i'm just of the opinion that making traitors less creative on average is a bad idea.

**Comment by ATH1909** [comment]
> Lol, trying to spin this as me centering the job around the antag experience is a creative take.

I was just pointing out that "The main criticism of Psychologist so far has been the lack of mechanics and unique content, here is one simple way of giving the job something unique as well as perfectly fitting, and now the sky is falling apparently. Curious." wasn't a very good argument.

**Comment by uomo91** [comment]
Alrighty, so the Psych job getting any unique content, worst of all a traitor item, would be unfairly locking the rest of the crew out of fun mechanics they should all be able to enjoy.

Thanks for your logical and well thought out feedback I guess.

**Comment by ATH1909** [comment]
> Alrighty, so the Psych job getting any unique content, worst of all a traitor item, would be unfairly locking the rest of the crew out of fun mechanics they should all be able to enjoy.
> 
> Thanks for your logical and well thought out feedback I guess.

It's not that they're "getting" unique content, it's that you're making **existing** traitor content traitor psychologist-only.

**Comment by itseasytosee** [comment]
Make a new traitor item that isn't held dearly by a regardable amount of people.
You seem as though you have some sort of hidden agenda. As if this is all a means to indirectly bring something you don't like about the hypnoflash to light

**Comment by uomo91** [comment]
> > Alrighty, so the Psych job getting any unique content, worst of all a traitor item, would be unfairly locking the rest of the crew out of fun mechanics they should all be able to enjoy.
> > Thanks for your logical and well thought out feedback I guess.
> 
> It's not that they're "getting" unique content, it's that you're making **existing** traitor content traitor psychologist-only.

You explicitly stated and just reiterated that your underlying huge fear about Psychologist was that them getting unique content means that's content which the rest of the crew wouldn't be able to engage with. (perish the thought of a job having things only they are supposed to/can do, almost like that's kind of why jobs exist to begin with)

> Make a new traitor item that isn't held dearly by a regardable amount of people.
You seem as though you have some sort of hidden agenda. As if this is all a means to indirectly bring something you don't like about the hypnoflash to light

I like the hypnoflash a lot, if there's anything I dislike about it it's how difficult it is to use successfully in contrast to how relatively benign it is. This wouldn't address that, it just puts it in the hands of a job with perfect RP justification for having it as well as the means to put it to good use. Sort of like... the other existing role-restricted uplinks items.

**Comment by itseasytosee** [comment]
Having a phycologist who drugs and hypnotizes their patients seems like a gimmick that could work perfectly well without them being the only ones to have the hypnoflash

**Comment by Shadowflame909** [comment]
The core issue undeniably is that your seizing what everyone had access to and making it psychologist only.

Pretty sure if one day you go, "I think the psychologist needs a pet-treatment option" so you take runtime from the CMO and put it in the Psychologist's office, pet-carrier and all. There'd be similar grievances of uplifting and moving items to add to the psychologist.

**Comment by uomo91** [comment]
> Having a phycologist who drugs and hypnotizes their patients seems like a gimmick that could work perfectly well without them being the only ones to have the hypnoflash

Why are His Grace and gorilla cubes and other things role restricted?
Could it be, because they fit perfectly and make sense to be?
The hypnoflash predating the existence of Psych doesn't really negate the fact that this would follow a sensical trend of what items are role-restricted, to who, and why.

**Comment by itseasytosee** [comment]
You keep saying why not and the people cry back why.

**Comment by Iamgoofball** [comment]
i was expecting this to be an XDTM PR not gonna lie

**Comment by Learner10** [comment]
If the issue is that it's hard to make the hypnoflash work, then should the traitor just acquire another piece of gear that'll make it very easy to use the hypno? Like say... a sleepy pen? Or some Gargle Blaster?

Ultimately, I'd settle up to only reserving the hypnoflashbang for the Psychologist.

Sure, a hypnoflash is perfect for a Psychologist, but it is also a very beloved gimmick item for many a player. Perhaps you could instead make an item that they can use that'll help give them reason to bring people into isolation. Or why not something like a camera plush that you can implant things into.

**Comment by wesoda25** [comment]
hypnoflashbang is just a bad idea and should be removed

**Comment by Spepbuthewranglesgithub** [comment]
this just feels like a hypnoflash removal with extra steps, seriously, how many people use the hypnoflash **_ALREADY_**, doing this would make it SO unused that we would probably have a removal PR a month later because it only got used a whole 10 times.

Who actually plays psychologist, really.

 chaplains have pray based fun, lawyers get to mess with sec at every turn, and curators... get to explore space easily? curators do need a rework... _but_, its still more content than the psychologist who gets to wander around medbay, being absolutely useless, do absolutely nothing of note, and get absolutely nobody in his office due to nobody caring about his existance.


to sum it up, give the psych more actual gameplay before you start giving him unique traitor items, **especially** already niche items that will probably end up going the way of the haunted magic eightball or get removed.

**Comment by Shadowflame909** [comment]
What spep said.  Make Psychologists traitors be the only ones who gets the hypoflash? A new meta will form where assistants go into the Psychologist's office purely for the antag-lite conversion and will stop interacting with them once they realize they dont have it.

It's how robotics is. But at least the roboticist can entertain themselves with mech building. All the psychologist will get once people realize they aren't a traitor this round is ghosted.

**Comment by nikothedude** [comment]
You have to answer one question first: Why is the hypnoflash bad (not underpowered balance-wise) in the hands of anyone else besides the psychologist?

**Comment by uomo91** [comment]
> You have to answer one question first: Why is the hypnoflash bad (not underpowered balance-wise) in the hands of anyone else besides the psychologist?

I actually don't have to answer this question since it wasn't something I stated to begin with.

**Comment by necromanceranne** [comment]
People don't go to robo to self-antag as borgs, what the fuck?

**Comment by Shadowflame909** [comment]
> 
> 
> People don't go to robo to self-antag as borgs, what the fuck?
Maybe I over-stated the prevelance of that mentality considering being a borg is all-right.

But on a traitor round, an emag is a common tool of craft, and one traitor only needs to have it for you to become antag lite.

Same philosophy here.

Also since I'm commenting again. I just want to say that:
>Why are His Grace and gorilla cubes and other things role restricted?
>Could it be, because they fit perfectly and make sense to be?

As far as I can see those items were made with the purpose of being role-restricted, and being job specific. This wasn't. With said creator of those items not agreeing with this change.

**Comment by HomocidlePancakes** [comment]
Let’s dissect the PR body one by one:

>The hypnoflash is not a very commonly purchased item

1)Provide proof
2)Low purchase rate can be a symptom of the item being shit, but it’s not the case here. The item has defined uses and is pretty well fleshed out overall.
3)You justify this change with low use rate, but the change will make the item used even less. Self-contradiction.

> It's a liability and a bit of monkeying around to even try hypnoflashing somebody, and if you succeed it's a further liability on if the person you flashed will actually play along and obey or just rat you out, or screw up your plan/be incompetent if they do obey. If your objective is greentexting at all cost or murderboning, chances are you will not touch this item with a ten foot pole.

What the fuck is this even? Your change boils down to making the item less available. This part does not address it at all and instead goes on some unrelated rant about people being incompetent and MuRdErBoNeRs.

>What the hypnoflash IS great for, is memes, funny things, and roleplay. And so, who better to give this item to than an RP job which attracts those type of players to begin with? In particular one for who it makes perfect RP sense for them to be hypnotizing people.

Again, your change boils down to lowering the accessibility on the flash. Instead of explaining to us why it’s a good change you go “well I think it would be cool if psychiatrist had it”. Come the fuck on.

> I feel this makes at least as much sense as the current role-restrictions on the brainwashing surgery disk.

1)Was intended to be role-restricted from the start
2)It’s a more effective alternative. Makes sense for it to be less available than the flash. However, you want to make the shittier option harder to get, which is dumb.

tl;dr The change is about making the flash less available. You haven’t provided a single point to justify it, instead spewing something about roleplay and fun. :thumbsdown: from me.

**Comment by cacogen** [comment]
katherine no

**Comment by XDTM** [comment]
Honestly i agree with the sentiment that psychologists and hypnosis go hand in hand. 

I think it would be much less controversial if it was a positive reinforcement towards the psychologist side (something like a discount, or a role-specific more powerful version of the same item) instead of negative over the other roles, with restriction.

**Comment by MrPerson** [comment]
Positive reinforcement like that can't be done because then the weaker/more expensive version is too weak/expensive to actually see any use.

**Comment by ATH1909** [comment]
> Additionally quite frankly the "controversy" doesn't actually exist or matter even if it did, so there's no need to design around such a thing.

image

> multiple multi-paragraph long explanations from different people explaining why this change is a bad idea

🤔 

> I think it would be much less controversial if it was a positive reinforcement towards the psychologist side (something like a discount, or a role-specific more powerful version of the same item) instead of negative over the other roles, with restriction.

Maybe the psychologist-only version of the hypnotic flash could be a pocket watch that'd be like the hypnotic flash, but it wouldn't make the distinctive flashing noise or the burst of pink light when used?

**Comment by HomocidlePancakes** [comment]
> Additionally quite frankly the "controversy" doesn't actually exist or matter even if it did, so there's no need to design around such a thing.

>ahaha they just mad lol ignore them ahha

**Comment by cacogen** [comment]
yeah just make it cheaper or stronger for the psychologist

>Positive reinforcement like that can't be done because then the weaker/more expensive version is too weak/expensive to actually see any use.

I don't see how it being cheaper or stronger for the psychologist negates its usefulness for other jobs.

**Comment by Pugie** [comment]
Yeah thank you, this item is cancer

**Comment by Akrilla** [comment]
https://pastebin.com/2Miwddrn

for anyone curious about how popular the hypnoflash/bang is. This is from the last 21k rounds

**Comment by cacogen** [comment]
Are those numbers rounds or purchases?

**Comment by Akrilla** [comment]
The number is how many times it has been purchased

**Comment by ATH1909** [comment]
It's 55th in the list of 195 traitor items.

Wait, is that a list of just traitor purchases, or does it factor in nuke op purchases/nuke op-only items too?

**Comment by ishitbyabullet** [comment]
> Wait, is that a list of just traitor purchases, or does it factor in nuke op purchases/nuke op-only items too?

It seems to do so

**Comment by Learner10** [comment]
> 
> 
> https://pastebin.com/2Miwddrn
> 
> for anyone curious about how popular the hypnoflash/bang is. This is from the last 21k rounds

Now that's interesting info. Interestingly enough though, it seems that hypnotic grenades sit at the very bottom of the list, while hypnoflashes are on the upper end of the spectrum. So they're clearly popular.

**Comment by Tlaltecuhtli** [comment]
its 21k rounds which is like more than a year? and hypno flashbang were added  recently

**Comment by XDTM** [comment]
Yeah the grenades aren't accurate since they weren't available for most of the recorded rounds there

**Comment by ATH1909** [comment]
> > https://pastebin.com/2Miwddrn
> > for anyone curious about how popular the hypnoflash/bang is. This is from the last 21k rounds
> 
> Now that's interesting info. It seems that hypnotic grenades sit at the very bottom of the list, while hypnoflashes are on the upper end of the spectrum. So the flashes are clearly popular, while the grenades... well no one even bats an eye at them.

Hypnobangs have been in the game for barely a month, while some of those other traitor items have been in the game for years (maybe even a decade, if I have my timeline right).

**Comment by Akrilla** [comment]
> It's 55th in the list of 195 traitor items.
> 
> Wait, is that a list of just traitor purchases, or does it factor in nuke op purchases/nuke op-only items too?

It includes dynamic rounds, so it'll be nukies as well.

**Comment by Akrilla** [comment]
Two months: https://pastebin.com/yfewgJKB

**Comment by Coastfront** [comment]
Making an item that isnt purchased frequently locked behind another roleplay job is just going to make it curator magic ball 2 electric boogaloo, meaning they wont fucking buy it. If a niche item isnt used frequently making its use even more limited to MAYBE A SINGLE PERSON who MIGHT NOT BUY IT is just going to mean nobody is going to fucking buy it 85.5% of the time. I atleast see it SOMETIMES on engi or cmo traitors. Just make a new role restricted item for psychologist instead of making a pointless restriction on an already rare item.

**Comment by Coastfront** [comment]
Hmm yes in order to make this item that is already used frequently but in comparison to other items not used more restricted in who can use it to make it used more. Making it restricted is just going to mean nobody who wants to use it, who has an easier time using it can use it. While a job that is arguably one of the worst for using it gets it. Just make a better version of it available restricted for psychologist.

**Comment by SkeletalElite** [comment]
No one uses the hypnoflash/bang therefore lets role restrict it so even less people get the opportunity to use it.

**Comment by SpaceManiac** [comment]
While the thematic synergy is amusing, restricting the item where it was not previously restricted does not seem necessary to me.
